app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/data/geometries/PointGeometry.java

```diff
@@ -1,13 +1,16 @@
 package com.teamagam.gimelgimel.app.view.viewer.data.geometries;
 
+import android.os.Parcel;
+import android.os.Parcelable;
+
 import com.google.gson.annotations.SerializedName;
 
 /**
  * Simple data-object for latitude/longitude location
  */
-public class PointGeometry implements Geometry {
+public class PointGeometry implements Geometry, Parcelable {
```

Comment:
same comment as LocationSample:

I know you spent some time on it but I would use Android Studio's basic Parcelable implementation.